app/controllers/declaration/OriginationCountryController.scala

```diff
@@ -59,10 +59,10 @@ class OriginationCountryController @Inject()(
         formWithErrors => Future.successful(BadRequest(originationCountryPage(mode, formWithErrors))),
         validCountry => {
           val updatedCache =
-            request.cacheModel.locations.destinationCountries.getOrElse(DestinationCountries.empty()).copy(countryOfDispatch = validCountry)
+            request.cacheModel.destinationCountries.copy(countryOfDispatch = validCountry)
```

Comment:
please move to method on `destinationCountries`

Reply:
Done